lib/tasks/database.rake

```diff
@@ -14,7 +14,7 @@ namespace 'db:sessions' do
   desc "Trim old sessions from the table (default: > 30 days)"
   task :trim => [:environment, 'db:load_config'] do
     cutoff_period = (ENV['SESSION_DAYS_TRIM_THRESHOLD'] || 30).to_i.days.ago
-    ActiveRecord::SessionStore::Session.
+    ActiveRecord::SessionStore::ActiveRecordStore.session_class.
       where("updated_at < ?", cutoff_period).
       delete_all
```

Comment:
I think this will not work for the general case. The requirements for `ActiveRecord::SessionStore::ActiveRecordStore.session_class` are pretty minimal, see https://github.com/rails/activerecord-session_store/blob/master/README.md

> You must implement these methods:
>
> * `self.find_by_session_id(session_id)`
> * `initialize(hash_of_session_id_and_data, options_hash = {})`
> * `attr_reader :session_id`
> * `attr_accessor :data`
> * `save`
> * `destroy`

no `where`, no `delete_all`.